### DISCUSSION TITLE
Alpine stops making DOM updates

### DISCUSSION THREAD

**Discussion by rossb** [alpinejs/alpine#1397] [7. Bugs (archived)]
Bug description: all Alpine components break on page, failing to render visual updates to DOM (though the code executes and no error is reported in the console).

Alpine still executes its JavaScript code (DOM classes update, sound plays on videos, items are added to basket, etc), but the browser stops rendering any visual updates related to them in the DOM from Alpine. So for example a video sound plays, without showing the video.

Cause: seems to happen when either `<video>` elements are on the page, or when the JavaScript package `@shopify/theme-product-form` is included on the page, and specifically when a call related to that package is made within any kind of async callback (whether async function, XMLHttpRequest, or fetch). When included on the page and called synchronously, the bug does not occur.

The bug consistently happens on mobile devices, and with low RAM available. On devices with high RAM available it happens less often, or not at all.

The bug affects all user agents we have tested on -- the latest version of Google Chrome (88.0.4324.150 is my current version), Mobile Safari (iOS 14.4), and Mac Firefox (85.0.2) are three examples which all show the bug in the same way.

**Comment by ryangjchandler** [comment]
@rossb - Are you able to reproduce in a CodePen?
